investigating-with-grr/flows/specifying-file-paths.md

```diff
@@ -103,11 +103,27 @@ Will match:
 /root/1/2/3/another.doc
 ```
 
-More depth can be specified by adding a number to the \*\*, e.g. this
+More depth can be specified by adding a number to the `**, e.g. this
 performs the same search 10 levels deep:
 
 ```docker
-/root/**10.doc
+/root/**10/*.doc
+```
+
+It can also be combined with interpolation.
+
+The following statement:
+
+```docker
```

Comment:
I have no idea why the existing documentation used `docker` as syntax highlighting but I would not add new blocks with it. I think plain triple backtackticks is good enough here.

Reply:
done